### DISCUSSION TITLE
Multiple surveys in a base project

### DISCUSSION THREAD

**Discussion by mauricioranzan** [opengisch/QField#2919] [Q&A]
Hi!
How can I do multiple surveys using a common base project?
For example: I have a base project that I use for various services.
Today I make copies and delete everything from the database to start a new service.
Is there an easier or correct way to do this?

**Comment by OBVYamaska** [comment]
It is surely possible. We maintain a project that hosts about 20 survey forms, with various options to filter or combine them.
It all work great. That being said, it can seriously slow down QField, and won't work with the actual version of QFieldCloud, which has a download limit (this is supposed to go away in the future).

What have you tried and what are your concerns?

**Reply by mauricioranzan** [reply]
What I would like to do is start each survey as a new project, empty layers, no features.
Currently I have to do this manually, creating new projects with different names.
But I would like to find an alternative to facilitate this in QField Cloud.
